src/geometry/planar.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Finding faces of a planar graph
+tags:
+  - Translated
+e_maxx_link: facets
+---
+# Finding faces of a planar graph
+
+Consider a graph $G$ with $n$ vertices and $m$ edges, which can be drawn on a plane in such a way that two edges intersect only at a common vertex (if it exists).
+Such graphs are called **planar**. Now suppose that we are given a planar graph together with its straight-line drawing, which means that for each vertex $v$ we have a corresponding point $(x, y)$ and all edges are drawn as line segments between these points without intersection (such drawing always exists). These line segments split the plane into several regions, which are called faces. Exactly one of the faces is unbounded. This face is called **outer**, while the other faces are called **inner**.
```

Comment:
I think, the common term in English literature for this is "planar embedding". See e.g. [here](mathworld.wolfram.com/PlanarEmbedding.html). I would suggest using this instead of "drawing" throughout the article to make it consistent with external resources.

Reply:
done